src/main/java/core/basesyntax/MainApp.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class MainApp {
+    public static void main(String[] args) {
+        List<Integer> list = new ArrayList<>();
```

Comment:
Usage of Java Collections Framework's List and ArrayList is not allowed as per the checklist. Consider implementing your own version of a list, or modify the task if the use of standard collections is essential.